bot/code_review_bot/tasks/clang_tidy.py

```diff
@@ -155,10 +155,18 @@ class ClangTidyTask(AnalysisTask):
 
     artifacts = ["public/code-review/clang-tidy.json"]
 
+    @property
+    def display_name(self):
+        return "clang-tidy"
+
+    def build_help_message(self, files):
+        files = " ".join(files)
+        return f"`./mach static-analysis check {files}` (C/C++)"
```

Comment:
I have been thinking about that. No need to do that here 
but I would be interested to tell the developer if the lint has an autofix or not
for example, mozlint/flake8 with --fix will call autopep8

Reply:
Only sane way to do that is if the help message comes from the task itself: https://github.com/mozilla/code-review/issues/339